keyboards/handwired/gvlsplit/readme.md

```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](https://i.imgur.com/X9vh2FI.png)
```

Comment:
```suggestion
![gvlsplit](https://i.imgur.com/X9vh2FIh.png)
```

Reply:
whats the difference here? was the previous url not working? both urls load the image for me